OtripleS.Web.Api.Tests.Unit/Services/Foundations/CalendarEntryAttachments/CalendarEntryAttachmentServiceTests.Exceptions.Add.cs

```diff
@@ -94,8 +94,11 @@ public async Task ShouldThrowServiceExceptionOnAddWhenExceptionOccursAndLogItAsy
             CalendarEntryAttachment someCalendarEntryAttachment = CreateRandomCalendarEntryAttachment();
             var serviceException = new Exception();
 
+            var failedCalendarEntryAttachmentService =
```

Comment:
failedCalendarEntryAttachmentServiceException should be the variable name. Please verify in all the places.